src/lib/app.js

```diff
@@ -74,14 +81,29 @@ export async function openLinkInternal(url) {
  * @return {Promise<any>}
  */
 export async function openNativeFileDialog() {
+  log.debug('app/openNativeFileDialog');
   return sendMessage(cmd.nativeUpload);
 }
 
+/**
+ * Get all language info from the app.
+ *
+ * Shoud get split up in the future and
+ * removed from the public interface
+ *
+ * @return {Promise<any>}
+ */
+export async function getLanguageInfos() {
+  log.debug('app/getLanguageInfos');
+  return sendMessage(cmd.langInfos);
```

Comment:
this seems to be the same as "getContentLnaguages". Also the commend seems strange to me